src/test/resources/application.properties

```diff
@@ -12,3 +12,5 @@ oauth2.secret.key=thisIsTheSecretKeyForTest-Etl001
 spring.data.mongodb.host=localhost
 spring.data.mongodb.port=27017
 spring.data.mongodb.database=ea_restaurant_java_etl
+
+password.encoding.type=BASE64
```

Comment:
maybe ``security.password.encoding.type` is more specific?

Reply:
Done.